src/main/java/com/google/step/snippet/external/StackOverflowClient.java

```diff
@@ -39,9 +39,19 @@
   private static final String ANSWER_ID_PARAMETER = "answer_id";
   // Set 200 to be the maximum length of description for MVP.
   private static final int MAX_DESCRIPTION_LENGTH = 200;
+  private String CSE_ID;
 
+  public StackOverflowClient(String CSE_ID) {
+    this.CSE_ID = CSE_ID;
+  }
+
+  @Override
+  public String getCSEId() {
+    return CSE_ID;
+  }
   /* This method build the desired card after getting each field. */
```

Comment:
nit: This method _builds_ the desired card...

It'd be nice to have more formal JavaDoc:

```java
/**
  * Creates and returns a {@code Card} for the given StackOverflow question URL.
  * @param url the URL of the StackOverflow question to create a card for
  * @return the created card, or {@code null} if a card could not be created
  */